agenthub/README.md

```diff
@@ -70,3 +91,50 @@ def step(self, state: "State") -> "Action"
 
 `step` moves the agent forward one step towards its goal. This probably means
 sending a prompt to the LLM, then parsing the response into an `Action`.
+
+## Agent Delegation
+
+        OpenDevin is a multi-agentic system. Agents can delegate tasks to other agents, whether
```

Comment:
Should we remove the extra indentation here?

Reply:
Fixed